src/modules/new-request-form/data-types/Field.ts

```diff
@@ -1,12 +1,12 @@
 export interface Field {
+  id: number;
   name: string;
-  value: string;
+  value?: string | boolean;
```

Comment:
Couldn't agree more and I think in the end it would be best to end up with something like `type Field = BaseField | DueDateField | CheckboxField | AttachmentsField | ...` 👍🏼 

> If we need to merge this PR ASAP to solve the issues with the new data format, we can revisit the typings in a following PR

I thought it would be premature to do this now when we still haven't decided the final shape of all `Field`s is.
If it's ok with you let's take it in a follow up PR.